src/package/packageVersionCreate.ts

```diff
@@ -244,31 +258,33 @@ export class PackageVersionCreate {
     packageVersBlobZipFile: string
   ): Promise<PackageVersionCreateRequest> {
     const zipFileBase64 = fs.readFileSync(packageVersBlobZipFile).toString('base64');
-    const requestObject = {
+    const partialRequestObject: Partial<PackageVersionCreateRequest> = {
       Package2Id: this.packageId,
       VersionInfo: zipFileBase64,
       Tag: this.options.tag,
       Branch: this.options.branch,
       InstallKey: this.options.installationkey,
       Instance: this.options.buildinstance,
       SourceOrg: this.options.sourceorg,
-      CalculateCodeCoverage: this.options.codecoverage || false,
-      SkipValidation: this.options.skipvalidation || false,
+      CalculateCodeCoverage: this.options.codecoverage ?? false,
+      SkipValidation: this.options.skipvalidation ?? false,
       // note: the createRequest's Language corresponds to the AllPackageVersion's language
       Language: this.options.language,
     };
 
     // Ensure we only include the Language property for a connection api version
     // of v57.0 or higher.
     if (this.connection.getApiVersion() < '57.0') {
-      if (requestObject.Language) {
+      if (partialRequestObject.Language) {
         this.logger.warn(
-          `The language option is only valid for API version 57.0 and higher. Ignoring ${requestObject.Language}`
+          `The language option is only valid for API version 57.0 and higher. Ignoring ${partialRequestObject.Language}`
         );
       }
-      delete requestObject.Language;
+      delete partialRequestObject.Language;
     }
 
+    const requestObject = partialRequestObject as PackageVersionCreateRequest;
```

Comment:
what makes the ` as` necessary?

Reply:
Type PackageVersionCreateRequest does not agree with how its used. Adjusting the type.